codebuild/linux-gcc-7x-x64.yml

```diff
@@ -12,7 +12,7 @@ phases:
   build:
     commands:
       - echo Build started on `date`
-      - ./codebuild/common-linux.sh
+      - ./codebuild/common-linux.sh -DCMAKE_BUILD_TYPE="RelWithDebInfo"
```

Comment:
Should we just move this flag to the common-* scripts?

Reply:
no i'm gonna have to add debug builds too.